src/main/java/nextstep/courses/domain/Course.java

```diff
@@ -13,13 +17,19 @@ public class Course {
 
     private LocalDateTime updatedAt;
 
+    private List<Session> sessions = new ArrayList<>();
+
```

Comment:
```suggestion
    private List<Session> sessions;

    private Course(Long id, String title, Long creatorId, LocalDateTime createdAt, LocalDateTime updatedAt, List<Session> sessions) {
        this.id = id;
        this.title = title;
        this.creatorId = creatorId;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.sessions = sessions;
    }

    public Course(Long id, String title, Long creatorId, LocalDateTime createdAt, LocalDateTime updatedAt) {
        this.id = id;
        this.title = title;
        this.creatorId = creatorId;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.sessions = new ArrayList<>();
    }
    
```

저는 생성자를 다양하게 사용하기 위해
초기화는 생성자에서 해줄것 같아요. 

생성자가 다양하면, 테스트 할때
`assSession` 같은 메소드를 사용하지 않고도 
바로 `Course`를 생성할 수 있는게 장점인것 같아요.